sklearn/model_selection/_search.py

```diff
@@ -999,3 +1004,356 @@ def fit(self, X, y=None, labels=None):
                                           self.n_iter,
                                           random_state=self.random_state)
         return self._fit(X, y, labels, sampled_params)
+
+
+class SequentialSearchCV(BaseSearchCV):
+    """Search over hyperparameters using Gaussian process.
+
+    The parameters of the estimator used to apply these methods are optimized
+    by cross-validated search over parameter settings.
+
+    In contrast to GridSearchCV, not all parameter values are tried out, but
+    rather a fixed number of parameter settings are sampled from the specified
```

Comment:
I think it should be clearer (perhaps only for those of us for whom "acquisition function" is non-standard terminology) that the acquisition function selects the next parameter settings given previous CV evaluations.